lib/developer_portal/lib/liquid/tags/stripe_form.rb

```diff
@@ -5,7 +5,9 @@ class StripeForm < Liquid::Tags::PaymentGatewayBaseForm
 
       desc "Renders the stripe form"
       def render(context)
-        render_erb context, "payment_gateways/stripe", text: @text
+        account = context.registers[:site_account]
+        template = account.provider_can_use?(:stripe_elements) ? 'payment_gateways/stripe_elements' : 'payment_gateways/stripe'
```

Comment:
Maybe this should have a unit test of its own 🤔

Reply:
Yes